apps/graphql/src/dataloaders/itinerariesHelpers.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+
+import type {
+  ApiRouteItemType,
+  ProviderTypeType,
+} from './ItinerariesloaderTypes';
+
+export const getVehicle = (type: ?ProviderTypeType, uniqueNo: string) => ({
+  type: type ?? null,
+  uniqueNo: uniqueNo ?? null,
+});
+
+export const getProvider = (provider: ?string) => ({
+  id: 'ads',
+  name: provider ?? null,
+});
+
+export const getItineraryType = (routes: Array<Array<string>>) => {
+  if (routes == null) {
+    return null;
+  }
+  if (routes.length === 1) {
+    return 'oneway';
+  }
+  if (routes.length === 2 && routes[0][0] === routes[1][1]) {
+    return 'return';
+  }
+  return 'multicity';
+};
+
+export const getSectors = (
+  routesList: Array<ApiRouteItemType>,
+  routesMap: Array<Array<string>>,
+): Array<Array<ApiRouteItemType>> => {
+  let routesSubList = routesList;
+  return routesMap.map(routeMap => {
+    const routeListEndSectorIndex = routesSubList.findIndex(
+      routeItem => routeItem.flyTo === routeMap[1],
+    );
+    const tempRoutesSubList = routesSubList;
+    routesSubList = tempRoutesSubList.slice(routeListEndSectorIndex + 1);
+    return tempRoutesSubList.slice(0, routeListEndSectorIndex + 1);
+  });
+};
+
+export const getDate = (local: ?Date, utc: ?Date) => ({
+  local: local ?? null,
+  utc: utc ?? null,
+});
+
+export const getCountry = (name: ?string, code: ?string) => ({
+  id: 'aaa',
+  name: name ?? '',
+  code: code ?? '',
+  slug: 'slug',
+  flagURL: 'flagUrl',
+});
+
+export const getLocation = (
+  locationId?: string,
+  name?: string,
+  countryName?: string,
+  countryCode?: string,
+) => ({
+  id: '',
```

Comment:
I have been thinking about and maby we can avoid this `locationId` and instead of this use just an `id`.
It will be encoded by GlobalID and if we need it we can decode it any time.